thorlcr/activities/lookupjoin/thlookupjoinslave.cpp

```diff
@@ -2183,7 +2183,10 @@ class CHTBase
     }
     void setup(CSlaveActivity *activity, rowidx_t size, IHash *_leftHash, IHash *_rightHash, ICompare *_compareLeftRight)
     {
-        size32_t sz = sizeof(const void *)*size;
+        unsigned __int64 _sz = sizeof(const void *)*size;
```

Comment:
Sorry Jake, going to be really picky...  If size_t is 32bit this multiply will be done as a 32bit operation - which means the test won't work.  At least one of the arguments needs to be cast to unsigned __int64 first!